### PR TITLE
HPCC-18643 int64 (ESDL) translated to integer8 (ECL)

### PR DESCRIPTION
**Pull request by rpastrana**
- Maps ESDL int64 types to integer8 in ECL

Signed-off-by: Rodrigo Pastrana <rodrigo.pastrana@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [ ] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by rpastrana** [comment]
@shamser please verify this xslt change fixes the issue you reported. This file can be found here: 
/opt/HPCCSystems/componentfiles/xslt/esdl2ecl.xslt

@afishbeck quick code review please.

File: esp/xslt/esdl2ecl.xslt [hunk 1]
```diff
@@ -229,6 +229,7 @@ end;
 		<xsl:when test="$basic_type='int'"><xsl:text disable-output-escaping="yes">integer</xsl:text><xsl:if test="not(@ecl_max_len)"><xsl:value-of select="$size"/></xsl:if></xsl:when>
 		<xsl:when test="$basic_type='long'"><xsl:text disable-output-escaping="yes">integer4</xsl:text><xsl:if test="not(@ecl_max_len)"><xsl:value-of select="$size"/></xsl:if></xsl:when>
 		<xsl:when test="$basic_type='short'"><xsl:text disable-output-escaping="yes">integer2</xsl:text></xsl:when>
+        <xsl:when test="$basic_type='int64'"><xsl:text disable-output-escaping="yes">integer8</xsl:text></xsl:when>
```
Comment (afishbeck) [inline esp/xslt/esdl2ecl.xslt]: Indentation looks off.

**Review by afishbeck — COMMENTED** [review]
Trivial comment but looks good otherwise.

**Comment by afishbeck** [comment]
@rpastrana back to you

File: esp/xslt/esdl2ecl.xslt (hunk 1 above)
Comment (rpastrana) [reply esp/xslt/esdl2ecl.xslt]: it is, I used spaces but the file uses tabs. I set up my editor to automatically convert new tabs to spaces. Should I force tabs in order to maintain the current format?

**Comment by rpastrana** [comment]
Spacing changed to match existing file format

**Review by afishbeck — APPROVED** [review]
@rpastrana looks good.

The spacing didn't matter too much I guess, but consistency is good. Thanks.

**Review by shamser — APPROVED** [review]
@rpastrana Code reviewed. Confirmed fix resolved original issue.  All good.

**Comment by rpastrana** [comment]
@afishbeck @shamser thanks for the code review.

**Pull request merged**